### ISSUE TITLE
Add TriplesMap property to IMapBase

### ISSUE THREAD

**Issue by mchaloupka** [r2rml4net/r2rml4net#24]
It would be really beneficial if it would be possible to back reference the whole triplesmap from a single mapping (represented by `IMapBase`). It seems like a small change as the value is already available at `BaseConfiguration` type.

**Comment by tpluscode** [comment]
Essentially, making [TriplesMap](https://github.com/r2rml4net/r2rml4net/blob/master/src/TCode.r2rml4net/Mapping/Fluent/BaseConfiguration.cs#L126) public

**Comment by mchaloupka** [comment]
Exactly, I am now accessing the property using reflection. However, I am not able to mock the interface easily and allow the reflection at the same time. It seems to me like a really small change. If you do not have time, I can create a pull request for you.

**Comment by tpluscode** [comment]
Sure, fell free to submit a PR :)

**Comment by mchaloupka** [comment]
I have created the pull request.
https://github.com/r2rml4net/r2rml4net/pull/25

By the way, the project is not compilable under VS 2017. It would be nice to fix it, but it would require to change targets of the project (at least .NET 4.5).

**Comment by tpluscode** [comment]
I don't think it will be necessary to drop old .NET targets. On the other hand I'm fine with migrating .NET Standard and never looking back :). Thoughts?

**Comment by tpluscode** [comment]
Hey @mchaloupka. I'm afraid I won't be able to immediately fix this and deploy before my vacations (starting tomorrow). I will take care of #26 after February 12th

**Comment by mchaloupka** [comment]
No pressure.

Is there any issue with the code I gave you or you just reopened it as you are not able to create nuget?
